supertokens_python/recipe/session/utils.py

```diff
@@ -428,7 +459,7 @@ def validate_and_normalise_user_input(
         )
 
     if anti_csrf is None:
-        anti_csrf = "VIA_CUSTOM_HEADER" if cookie_same_site == "none" else "NONE"
+        anti_csrf = "VIA_CUSTOM_HEADER"
```

Comment:
Merge the base branch into this one so that code changes from other PRs dont show here